tests/system_tests.py

```diff
@@ -468,6 +474,24 @@ def setUp_file_action(self, expanded_file_name):
         return expanded_file_name
 
 
+def p(path):
```

Comment:
I am not very convinced by this helper. The advantage of using `os.path.join` directly in the test cases is that the developer do not need to think about which slashes he should use, he just would type:

```
filename = os.path.join("$data_path", "pocIssue283.jpg") #Note the lack of slashes here

# To reduce the verbosity we could do
from os.path import join
filename = join("$data_path", "pocIssue283.jpg")
```

Every python developer with some minimum experience will know what `os.path.join` is used for. 

As I commented before, to use a function name like `p` is a bad idea in any programming language. We should always have meaningful names for the functions to help to the code readability.

Reply:
My problem with `os.path.join` is that my original intention for the test suite was to keep the required Python knowledge at a minimum and to retain as much of the "look and feel" of the bash scripts. Using `os.path.join` is of course the most Pythonic way of solving this problem, but that will imho only make it more complicated for developers who don't know Python.

Also, I like to keep the slashes inside the string, as the string then is obviously a path.

Reply:
Fair enough. Let's follow the original design idea. If we can keep working like that, I'm happy with the solution.